src/content/docs/configuration/providers.mdx

```diff
@@ -94,190 +81,196 @@ Daytona allows you to update an existing Provider, enabling you to apply the lat
         Choose a provider to update
         1 item
         ===
-        │ docker-provider
-        │ v0.0.0
+        docker-provider
+        v0.0.0
     ```
 
     ```text
         Provider docker-provider has been successfully updated
     ```
 
-## Managing Targets
-
-### Set a Target
+## Uninstall a Provider
 
-Daytona allows you to set a Target to use when managing Workspaces.
-
-**Prerequisite**
-
-- **At least [one Provider installed](#managing-providers).**
-
-    Providers are essential for managing and deploying your Workspaces, so make sure you have configured at least one.
+Daytona allows you to uninstall an existing Provider, helping you manage your Providers by removing those that are no longer needed.
 
-1. Run the following command to set a Target:
+1. Run the following command to uninstall a Provider:
 
     ```shell
-    daytona target set
+    daytona provider uninstall
     ```
 
-2. Select the appropriate Provider for the environment you want to deploy to.
+2. Select the Provider you want to uninstall from the list.
 
     ```text
-        Choose a provider
+        Choose a provider to uninstall
         1 item
         ===
-        │ docker-provider
-        │ v0.0.0
+        docker-provider
+        v0.0.0
     ```
 
-3. Select `New Target`.
-4. Enter a name for your Target.
-5. Enter the appropriate configuration options when prompted.
-
     ```text
-        Remote Password
-        >
-
-        Remote Port
-        >
+        Provider docker-provider has been successfully uninstalled
+    ```
 
-        Remote User
-        Note: non-root user required
-        >
+## Docker
 
-        Sock Path
-        > /var/run/docker.sock
+The Docker Provider allows Daytona to create Workspace projects on Docker containers. The Provider integrates Daytona with Docker, enabling you to manage Workspaces on a Docker instance. Daytona installs a Provider for Docker (`docker-provider`) by default and adds a default [Target](/configuration/targets#docker) using the Docker Provider, allowing the creation of Workspaces on your local machine.
```

Comment:
`The Docker Provider allows Daytona to create Workspace projects on Docker containers. The Provider integrates Daytona with Docker, enabling you to manage Workspaces on a Docker instance.`

=> `The Docker Provider allows Daytona to create Workspace projects as Docker containers`